_posts/7-31-2019--INTROPOST.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: INTROPOST
+author: Normandy Rivver
+categories: ['Missouri State University', 'Computer Science']
+tags: ['CSC 596', 'robotics', 'robotics-game', 'transformable-robot-game']
+
```

Comment:
Delete this blank line.

Reply:
I assume you mean the blank line above the ---- and below the ----